lib/p2p/Pool.ts

```diff
@@ -431,6 +451,7 @@ class Pool extends EventEmitter {
         connectedNodesInfo.push({
           nodePubKey: connectedPeer.nodePubKey!,
           addresses: connectedPeer.addresses,
+          lastAddress: connectedPeer.address,
```

Comment:
I actually don't think we want to share *our* `lastAddress` with other peers - let them connect and determine which address works for them. It's possible `lastAddress` is reachable for us but not fro the external internet. Plus if it's not an advertised address, we don't want to share it.

Reply:
Then we actually don't want to share it at all ? Then I'm setting undefined to `lastAddress` here

Reply:
Yeah I believe so.